### ISSUE TITLE
It's a little bit strange to refer to previous version in the reference section.

### ISSUE THREAD

**Issue by PaulJeong14** [w3c/json-ld-syntax#88]
This is simple editorial comment for the document.
JSON_LD11(previous version of JSON_LD11 WD) is in the I.1  Normative Reference like below.

> [JSON-LD11]
> JSON-LD 1.1. Gregg Kellogg. W3C. 11 September 2018. W3C Working Draft. URL:https://www.w3.org/TR/json-ld11/

Most of W3C Document don't refer to previous version in the reference section.
It might to be better to delete it from reference section to avoid confusion and for consistency with other documents.

**Comment by gkellogg** [comment]
Yes, as @davidlehn said, references are placed because they exist within terms used between the different specs. This will take some figuring out how to eliminate them from the spec itself.

**Comment by gkellogg** [comment]
Note that the discussion is about a spec referencing itself, but the title is about referencing a previous version.

The bits referencing the previous version are:

> **processing mode**
> The processing mode defines how a JSON-LD document is processed. By default, all documents are assumed to be conformant with JSON-LD 1.0 [JSON-LD]. By defining a different version using the `@version` member in a context, or via explicit API option, other processing modes can be accessed. This specification defines extensions for the `json-ld-1.1` processing mode.

and

> `@version`
> Used in a context definition to set the processing mode. New features since JSON-LD 1.0 [JSON-LD] described in this specification are only available when processing mode has been explicitly set to `json-ld-1.1`.

This aspect is debatable. The fact that specs shouldn't reference themselves, is not. I've created w3c/respec#1862 to resolve.

**Comment by PaulJeong14** [comment]
I'm sorry for the confusion of the issue title.
I meant the previous version is not the previous version (JSON LD 1.0) but the previous version of the WD(JSON 1.1). 
I'm OK if you delete self-referencing.
